lib/prop/leaky_bucket_strategy.rb

```diff
@@ -8,9 +8,10 @@ class << self
       def counter(cache_key, options)
         bucket = Prop::Limiter.cache.read(cache_key) || zero_counter
         now = Time.now.to_i
-        leak_amount = (now - bucket.fetch(:last_updated)) / options.fetch(:interval) * options.fetch(:threshold)
+        leak_rate = (now - bucket.fetch(:last_updated)) / options.fetch(:interval).to_f
```

Comment:
The problem is integer division here. `now - last_updated` is usually smaller than `interval` which results in `0 * threshold` -> 0